src/pages/staking/index.js

```diff
@@ -1,422 +1,668 @@
-    flex-direction: column;
+const ComparisonGrid = styled.div`
+  display: grid;
+  grid-column-gap: 3rem;
+  grid-auto-rows: minmax(64px, auto);
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-areas:
+    "solo-title saas-title pool-title"
+    "solo-rewards saas-rewards pool-rewards"
+    "solo-risks saas-risks pool-risks"
+    "solo-reqs saas-reqs pool-reqs"
+    "solo-cta saas-cta pool-cta";
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      "solo-title"
+      "solo-rewards"
+      "solo-risks"
+      "solo-reqs"
+      "solo-cta"
+      "saas-title"
+      "saas-rewards"
+      "saas-risks"
+      "saas-reqs"
+      "saas-cta"
+      "pool-title"
+      "pool-rewards"
+      "pool-risks"
+      "pool-reqs"
+      "pool-cta";
+  }
+
+  h4 {
+    color: #787878;
   }
 `
 
-const Option = styled.div`
-  border-radius: 32px;
-  border: 1px solid
-    ${(props) =>
-      props.isActive ? props.theme.colors.primary : props.theme.colors.text};
-  color: ${(props) =>
-    props.isActive ? props.theme.colors.primary : props.theme.colors.text};
-  box-shadow: ${(props) =>
-    props.isActive ? props.theme.colors.tableBoxShadow : `none`};
+const ColorH3 = styled.h3`
+  grid-area: ${({ color }) => {
+    switch (color) {
+      case "stakingGold":
+        return "solo-title"
+      case "stakingGreen":
+        return "saas-title"
+      case "stakingBlue":
+        return "pool-title"
+      default:
+        return ""
+    }
+  }};
+  color: ${({ theme, color }) => theme.colors[color]};
+`
+
+const ButtonContaier = styled.div`
   display: flex;
-  align-items: center;
-  padding: 1rem 1.5rem;
-  margin: 0.5rem;
-  cursor: pointer;
+  gap: 1rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+  }
 `
 
-const OptionText = styled.div`
-  font-size: 1.5rem;
-  line-height: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    font-size: 1rem;
-    font-weight: 600;
+const StyledButtonLink = styled(ButtonLink)`
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: 100%;
   }
 `
 
-const StakeContainer = styled.div`
-  margin: 0 auto;
-  max-width: ${(props) => props.theme.breakpoints.m};
-  display: flex;
-  flex-direction: column;
-  text-align: center;
+const StyledCard = styled(Card)`
+  justify-content: flex-start;
+  h3 {
+    font-weight: 700;
+    margin: 0 0 1rem;
+  }
 `
 
-const paths = [
+const benefits = [
   {
-    emoji: ":money_with_wings:",
-    title: <Translation id="page-staking-title-1" />,
-    description: <Translation id="page-staking-desc-1" />,
+    title: "Earn rewards",
+    emoji: "💰",
+    description:
+      "Rewards are given for actions that help the network reach consensus. You'll get rewards for batching transactions into a new block or checking the work of other validators because that's what keeps the chain running securely.",
```

Comment:
```suggestion
      "Rewards are given for actions that help the network reach consensus. You'll get rewards for running software that batches transactions into new blocks and checks the work of other validators because that's what keeps the chain running securely.",
```
Not sure this edit is the best copy but I feel like we should mention somewhere that staking is _running software_. We don't explain that at all in the "What is staking?" above. I don't think we should assume people have seen the "Run a node" page.